project.clj

```diff
@@ -1,10 +1,10 @@
-(defproject yleisradio/new-reliquary "1.0.1"
+(defproject yleisradio/new-reliquary "1.0.2"
```

Comment:
Changing the "only" dependency one major up is only a one patch up on our library? I would bump minor version.